### ISSUE TITLE
"Save As" dialog title bar leaves user unsure which command got them there

### ISSUE THREAD

**Issue by alankilborn** [notepad-plus-plus/notepad-plus-plus#14737]
### Steps to Reproduce (STR) the Issue

1. Execute _File_ > _Save As_ and a standard file-saving dialog window opens
2. Note the title bar is "Save As" (English localization) and also note the appearance of the rest of the dialog
3. Cancel the dialog
4. Execute _File_ > _Save a Copy As_ and a standard file-saving dialog window opens
5. Note the title bar is "Save As" (English localization) and also note the appearance of the rest of the dialog
6. Cancel the dialog

### Actual Behavior

After step 2 or step 5 I am unsure which command brought me there -- the resulting dialogs are exactly the same in each case, so I get no visual confirmation that I'm in the "right place".

Of course I know for the purposes of the above STR, but in general, since _Save As_ and _Save a Copy As_ are adjacent items in the _File_ menu, a "mouse stutter" may not select my true intention, and, once the saving dialog opens, I have no way of knowing which command brought me to that point.

### Expected Behavior
* I expect the title bar of the dialog to be "Save As" when I've selected _File_ > _Save As_ (it is so this is not a problem)
* I expect the title bar of the dialog to be "Save Copy As" when I've selected _File_ > _Save a Copy As_ (this is the PROBLEM area)

### Debug Information
Notepad++ v8.6.2   (64-bit)
Build time : Jan 14 2024 - 02:16:00
Path : C:\NotepadPlusPlus\Misc\_NoBackup\Releases\npp.8.6.2\npp.8.6.2.portable.x64\notepad++.exe
Command Line : -titleAdd="8.6.2 64-bit" -multiInst
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 11 Pro (64-bit)
OS Version : 22H2
OS Build : 22621.3155
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3)
    NppConverter (4.5)
    NppExport (0.4)
    PythonScript (2)

**Comment by AndiKiissel** [comment]
See  #11860

**Comment by alankilborn** [comment]
@AndiKiissel Ah, sorry, I forgot someone (you, actually) had made that other issue.  :-(

**Comment by alankilborn** [comment]
Duplicate of #11860 (as previously noted).
